opencog/atomspace/AtomSpace.cc

```diff
@@ -99,6 +99,156 @@ AtomSpace& AtomSpace::operator=(const AtomSpace&)
          "AtomSpace - Cannot copy an object of this class");
 }
 
+bool AtomSpace::compare_atomspaces(const AtomSpace& space_first,
+                                   const AtomSpace& space_second,
+                                   bool check_truth_values,
+                                   bool emit_diagnostics)
+{
```

Comment:
Well, it is not needed by multiple unit tests in multiple directories .. so, for now, put it in `/tests/persist/sql` and link it into those tests that need it.  If, for some reason, you have tests in `/tests/atomspace` tat need it, then maybe something fancier is needed. I dunno. Keep it simple.